### DISCUSSION TITLE
Use ClassificationStore Title in Query

### DISCUSSION THREAD

**Discussion by designjpp** [pimcore/pimcore#17167] [Data-Hub]
Hello, 

How can i use a title of classificationStore in Query ?

[ClassificationStore Query Docs](https://pimcore.com/docs/platform/Datahub/GraphQL/Query/Query_Samples/ClassificationStore)

**Comment by markus-moser** [comment]
Hi,
currently this does not work. You can only query the technical names of the attributes but not the title.
